internal/tools/k8s/README.md

```diff
@@ -1,21 +1,7 @@
 # Kubernetes Package
 
-This package contains the resources necessary to create a kubernetes client capable of interacting with Mattermost Custom Resources such as a cluster installation.
+This package contains the resources necessary to create a client capable of interacting with different kubernetes clusters.
 
-This package leverages code generated using https://github.com/kubernetes/code-generator
+The clientset for the Mattermost `ClusterInstallation` custom resource now resides in the [mattermost-operator](https://github.com/mattermost/mattermost-operator) repository. This provides a single source of truth for `ClusterInstallation` objects. To change the clientset used here, first update the code in `mattermost-operator` and then import the changes.
```

Comment:
To be precise, does `import the changes` mean update the Go module dependencies?

Reply:
Yeah, exactly